### PR TITLE
 Factorize the Min and Max value settings group 

### LINKED ISSUE #1927: Image enhancement description is repeated multiband, singleband and pseudocolor renderers
In the Style raster section of Working with raster, the contrast enhancement description is repeated several times. One for each of the renderer that uses it (multiband, singleband gray, singleband pseudocolor).

http://docs.qgis.org/testing/en/docs/user_manual/working_with_raster/raster_properties.html#style-properties

The text should be harmonized and put as a common section for several renderers.

### PR DESCRIPTION
**Pull request by DelazJ**
Currently setting the mix and max values description is repeated in the doc. This PR fixes this with a dedicated section and links from each render type that makes use of such setting.
A screenshot for the section is also added.
Fixes #1927

### CODE REVIEW

File: source/docs/user_manual/working_with_raster/raster_properties.rst [hunk 1]
```diff
@@ -300,6 +246,53 @@ defined color table for other sessions.
...
 greater than the :guilabel:`Max` value.
 
+
+.. _minmaxvalues:
+
+Setting the min and max values
+..............................
+
+By default, QGIS reports the :guilabel:`Min` and :guilabel:`Max` values of the band(s)
+of the raster. A lot of images have a few very low and high data which can have a
```
Comment (havatv) [inline source/docs/user_manual/working_with_raster/raster_properties.rst]:
```suggestion
of the raster. A few very low and/or high values can have a
```

File: source/docs/user_manual/working_with_raster/raster_properties.rst [hunk 2]
```diff
@@ -300,6 +246,53 @@ defined color table for other sessions.
...
+
+.. figure:: img/rasterMinMaxValues.png
+   :align: center
+
+   Raster Symbology - Min and Max Value Settings
+
+
+Available options are:
+
+* |radioButtonOff| :guilabel:`User defined`: Allows to manually override the default
```
Comment (havatv) [inline source/docs/user_manual/working_with_raster/raster_properties.rst]:
```suggestion
* |radioButtonOff| :guilabel:`User defined`: The default
```

File: source/docs/user_manual/working_with_raster/raster_properties.rst [hunk 3]
```diff
@@ -300,6 +246,53 @@ defined color table for other sessions.
...
+  :guilabel:`Min` and :guilabel:`Max` values of the band(s)
```
Comment (havatv) [inline source/docs/user_manual/working_with_raster/raster_properties.rst]:
```suggestion
  :guilabel:`Min` and :guilabel:`Max` values of the band(s) can be overridden
```

File: source/docs/user_manual/working_with_raster/raster_properties.rst [hunk 4]
```diff
@@ -300,6 +246,53 @@ defined color table for other sessions.
...
+* |radioButtonOff| :guilabel:`Cumulative count cut`: Removes the outliers data.
```
Comment (havatv) [inline source/docs/user_manual/working_with_raster/raster_properties.rst]:
```suggestion
* |radioButtonOff| :guilabel:`Cumulative count cut`: Removes outliers.
```

File: source/docs/user_manual/working_with_raster/raster_properties.rst [hunk 5]
```diff
@@ -300,6 +246,53 @@ defined color table for other sessions.
...
+  The standard data range of the data values is set from ``2%`` to ``98%`` and can
```
Comment (havatv) [inline source/docs/user_manual/working_with_raster/raster_properties.rst]:
```suggestion
  The standard range of values is ``2%`` to ``98%``, but can
```

File: source/docs/user_manual/working_with_raster/raster_properties.rst [hunk 6]
```diff
@@ -300,6 +246,53 @@ defined color table for other sessions.
...
+  be adapted manually. With this setting, the gray character of the image can disappear.
```
Comment (havatv) [inline source/docs/user_manual/working_with_raster/raster_properties.rst]:
```suggestion
  be adapted manually.
```

Comment (havatv) [reply source/docs/user_manual/working_with_raster/raster_properties.rst]: I am am not sure what is meant by the "gray character of the image".

File: source/docs/user_manual/working_with_raster/raster_properties.rst [hunk 7]
```diff
@@ -300,6 +246,53 @@ defined color table for other sessions.
...
+* |radioButtonOn| :guilabel:`Min/max`: Creates a color table with all of the
```
Comment (havatv) [inline source/docs/user_manual/working_with_raster/raster_properties.rst]:
```suggestion
* |radioButtonOn| :guilabel:`Min/max`: Uses the whole range of values in the image band.
```

File: source/docs/user_manual/working_with_raster/raster_properties.rst [hunk 8]
```diff
@@ -300,6 +246,53 @@ defined color table for other sessions.
...
+  data included in the image (e.g., QGIS creates a color table with 256 values,
```
Comment (havatv) [inline source/docs/user_manual/working_with_raster/raster_properties.rst]:
```suggestion
```

File: source/docs/user_manual/working_with_raster/raster_properties.rst [hunk 9]
```diff
@@ -300,6 +246,53 @@ defined color table for other sessions.
...
+  given the fact that you have 8 bit bands).
```
Comment (havatv) [inline source/docs/user_manual/working_with_raster/raster_properties.rst]:
```suggestion
```

File: source/docs/user_manual/working_with_raster/raster_properties.rst [hunk 10]
```diff
@@ -242,35 +211,12 @@ You can also create individual color maps for the single bands here.
    Raster Symbology - Singleband pseudocolor rendering
 
 
-Three types of color interpolation are available:
-
-#. Discrete
-#. Linear
-#. Exact
+Using a :guilabel:`Band` of the layer and its :ref:`values range set <minmaxvalues>`,
```
Comment (havatv) [inline source/docs/user_manual/working_with_raster/raster_properties.rst]:
```suggestion
Using a :guilabel:`Band` of the layer and a :ref:`values range <minmaxvalues>`,
```

File: source/docs/user_manual/working_with_raster/raster_properties.rst (hunk 6 above)
Comment (DelazJ) [reply source/docs/user_manual/working_with_raster/raster_properties.rst]: Me neither. This PR mainly reuses the text that was already there and this expression that i suspected to relate to pseudocolor gray is also available in multiband and singleband pseudocolor. Thus, I reuse ;)

File: source/docs/user_manual/working_with_raster/raster_properties.rst [hunk 11]
```diff
@@ -120,9 +120,21 @@ Multiband color
 
 With the multiband color renderer, three selected bands from the image will be
 rendered, each band representing the red, green or blue component that will be
-used to create a color image. You can choose several :guilabel:`Contrast
-enhancement` methods: 'No enhancement', 'Stretch to MinMax', 'Stretch and clip
-to MinMax' and 'Clip to min max'.
+used to create a color image. QGIS automatically fetches for each band its
```
Comment (havatv) [inline source/docs/user_manual/working_with_raster/raster_properties.rst]:
```suggestion
used to create a color image. QGIS automatically fetches
```

File: source/docs/user_manual/working_with_raster/raster_properties.rst (hunk 9 above)
Comment (DelazJ) [reply source/docs/user_manual/working_with_raster/raster_properties.rst]: Another widely reused expression I did not understand.

File: source/docs/user_manual/working_with_raster/raster_properties.rst [hunk 12]
```diff
@@ -120,9 +120,21 @@ Multiband color
...
+:guilabel:`Min` and :guilabel:`Max` values over the whole raster and scale
```
Comment (havatv) [inline source/docs/user_manual/working_with_raster/raster_properties.rst]:
```suggestion
:guilabel:`Min` and :guilabel:`Max` values for each band of the raster and scales
```

File: source/docs/user_manual/working_with_raster/raster_properties.rst [hunk 13]
```diff
@@ -120,9 +120,21 @@ Multiband color
...
+the coloring accordingly but you can control the values range with the help
```
Comment (havatv) [inline source/docs/user_manual/working_with_raster/raster_properties.rst]:
```suggestion
the coloring accordingly. You can control the value ranges with the help
```

File: source/docs/user_manual/working_with_raster/raster_properties.rst (hunk 6 above)
Comment (havatv) [reply source/docs/user_manual/working_with_raster/raster_properties.rst]: If we don't understand it, I guess we can safely remove it (it seems to be meant as an explanation).

File: source/docs/user_manual/working_with_raster/raster_properties.rst (hunk 9 above)
Comment (havatv) [reply source/docs/user_manual/working_with_raster/raster_properties.rst]: I don't think this adds any useful information, so I suggest we remove it.

**Pull request merged**